tests/suites/1_stateful/08_select_stage/08_00_parquet/08_00_02_select_files.sh

```diff
@@ -16,9 +16,12 @@ echo "create stage s2 url = '${DATADIR}' FILE_FORMAT = (type = PARQUET);"  | $MY
 cp "$CURDIR"/../../../../data/tuple.parquet ${DATADIR_PATH}/
 cp "$CURDIR"/../../../../data/sample.csv ${DATADIR_PATH}/
 
+echo "select * from infer_schema(location => '@s2/tuple.parquet');" | $MYSQL_CLIENT_CONNECT
```

Comment:
I prefer to add a single logic test for it, the progress like:
```
CREATE STAGE ss;
COPY INTO @ss FROM (SELECT number, number::VARCHAR FROM numbers(100)) FILE_FORMAT = (TYPE = PARQUET);
SELECT * FROM infer_schema(location => '@ss', pattern => '.*parquet')
```

Reply:
ok

Reply:
We need more types built here(int and varchar is for example):
```
SELECT number, number::VARCHAR FROM numbers(100)
```